lib/auth/clt.go

```diff
@@ -2601,6 +2601,35 @@ func (c *Client) SetAccessRequestState(ctx context.Context, reqID string, state
 	return nil
 }
 
+// GetPluginData loads all plugin data matching the supplied filter.
+func (c *Client) GetPluginData(ctx context.Context, filter services.PluginDataFilter) ([]services.PluginData, error) {
+	clt, err := c.grpc()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	seq, err := clt.GetPluginData(ctx, &filter)
+	if err != nil {
+		return nil, trail.FromGRPC(err)
+	}
+	data := make([]services.PluginData, 0, len(seq.PluginData))
```

Comment:
What's the rationale behind the copy? it seems like sq.Data could be returned as is?

Reply:
Required because we're casting the slice elements to `interface` values.